tests/test_smoke.py

```diff
@@ -1559,6 +1588,50 @@ def test_job_queue_with_docker(generic_cloud: str, image_id: str):
     run_one_test(test)
 
 
+@pytest.mark.do
+def test_do_job_queue_with_docker(generic_cloud: str, image_id: str):
```

Comment:
Is it possible to reuse the previous function and add another argument for accelerator to use? same for other tests in this file. You can checkout `@pytest.mark.parametrize`